tests/dotnet-e2e/dns/Program.cs

```diff
@@ -0,0 +1,6 @@
+using System.Net;
+
+var url = "https://google.com";
+Uri myUri = new Uri(url);
+var ip = (await Dns.GetHostAddressesAsync(myUri.Host))[0];
+Console.WriteLine(ip);

```

Comment:
II dont tthink we need to write to console? rather just exit the process, and do something along the lines of https://github.com/metalbear-co/mirrord/blob/main/tests/node-e2e/remote_dns/test_remote_dns_lookup_google.mjs#L19

Reply:
Absolutely. Let me make this change 👍🏻